### DISCUSSION TITLE
How to use golang grpc client send request to a grpc-web powered server?

### DISCUSSION THREAD

**Discussion by readthecodes** [grpc/grpc-web#1360] [Q&A - General]
now, i know use js to send request to grpc-web powered grpc server, but how can i use golang grpc client send request to a grpc-web powered server?

**Comment by sampajano** [comment]
@readthecodes Hi :)

My understanding is that what you call as "grpc-web powered grpc server" is a **grpc server** + **a proxy** (e.g. Envoy).

In that sense, you can make the golang grpc client to talk directly to that **grpc server**? :)
